docs/modules/stackablectl/pages/commands/cache.adoc

```diff
@@ -0,0 +1,85 @@
+= stackablectl cache
+
```

Comment:
Explain a bit more what is actually cached and why. Also maybe where?

Reply:
Will be added in a follow-up PR.